Header header

sensor_msgs/LaserScan laser
geometry_msgs/PoseStamped position
geometry_msgs/PoseStamped target
float32 minFrontDist
bool collision
